trunk/src/app/srs_app_http_stream.cpp

```diff
@@ -1075,6 +1094,7 @@ void SrsHttpStreamServer::http_unmount(SrsRequest* r)
 
     // Notify cache and stream to stop.
     if (stream->entry) stream->entry->enabled = false;
+    stream->expire();
     cache->stop();
 
     // Wait for cache and stream to stop.
```

Comment:
`stream->expire()` will disconnect the `SrsHTTPConn` soon, so no long to wait 120 seconds below. Maybe 10 seconds is OK.

Reply:
No essential difference. I want keep it.